zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/Interpreter.java

```diff
@@ -249,15 +249,22 @@ public Interpreter getInterpreterInTheSameSessionByClassName(String className) {
     private String name;
     //@SerializedName("interpreterClassName")
     private String className;
+    private boolean defaultInterpreter;
```

Comment:
Okay, thanks for explanation.

But previously, order of interpreter(default interpreter) in the interpreter group as well as order of interpreter group(default interpreter group) were controlled by `zeppelin.interpreters`. So user were able to select which interpreter in which group will be the default.

Now order of interpreter(default interpreter) is defined in interpreter-setting.json and can not be changed by user.
And order of interpreter group (default interpreter group) is defined in `zeppelin.interpreter.group.order`.

Anyway, after [alias for interpreter](https://issues.apache.org/jira/browse/ZEPPELIN-1012), i think all notebook can always binded to every interpreter settings (we can remove interpreter binding steps). then with combination of https://issues.apache.org/jira/browse/ZEPPELIN-707, i think default interpreter / interpretergroup will become no longer necessary.